test/helpers/kubectl.go

```diff
@@ -1978,12 +1978,12 @@ func (kub *Kubectl) ValidateServicePlumbing(namespace, service string) error {
 // ValidateKubernetesDNS validates that the Kubernetes DNS server has been
 // deployed correctly and can resolve DNS names. The following validations are
 // done:
-//  - The Kuberentes DNS deployment has at least one replica
-//  - All replicas are up-to-date and ready
-//  - All pods matching the deployment are represented by a CiliumEndpoint with an identity
-//  - The kube-system/kube-dns service is correctly pumbed in all Cilium agents
-//  - The service "default/kubernetes" can be resolved via the KubernetesDNS
-//    and the IP returned matches the ClusterIP in the service
+//   - The Kuberentes DNS deployment has at least one replica
+//   - All replicas are up-to-date and ready
+//   - All pods matching the deployment are represented by a CiliumEndpoint with an identity
+//   - The kube-system/kube-dns service is correctly pumbed in all Cilium agents
+//   - The service "default/kubernetes" can be resolved via the KubernetesDNS
+//     and the IP returned matches the ClusterIP in the service
```

Comment:
These aren't in the upstream commit, but then again it might not matter too much since it's changing a comment.

Reply:
i'll rebase them out.

Reply:
Tho, this makes the comment just end in 'done:', is that expected?

Reply:
Welp, I rebased it away. Gonna resolve this.

Reply:
I don't think the comment should be removed. The diff was just pointing out a difference in whitespace which wasn't there in the upstream commit.

Reply:
OH.

Reply:
my gofmt is doing that.